### PR TITLE
barrel vs ladder in roc level in 80x

### PR DESCRIPTION
**Pull request by gouskos**
add DQM plot for the pixel barrel.

### CODE REVIEW

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
-1
see also #13759

**Comment by gouskos** [comment]
Pull request is updated according to your instructions

**Comment by deguio** [comment]
please test

File: DQM/SiPixelMonitorTrack/src/SiPixelTrackResidualSource.cc [hunk 1]
```diff
@@ -894,6 +921,12 @@ void SiPixelTrackResidualSource::analyze(const edm::Event& iEvent, const edm::Ev
   //iEvent.getByLabel( clustersrc_, clusterColl );
   iEvent.getByToken( clustersrcToken_, clusterColl );
   auto const & clustColl = *(clusterColl.product());
+
+  // get digis
+  edm::Handle< edm::DetSetVector<PixelDigi> >  digiinput;
+  iEvent.getByToken( digisrcToken_, digiinput );
+  if (!digiinput.isValid()) { cout << "no digis found\n"; }
```
Comment (deguio) [inline DQM/SiPixelMonitorTrack/src/SiPixelTrackResidualSource.cc]: cout is not allowed. please use the message logger to raise a warning.
thanks,
F.

**Comment by deguio** [comment]
apart from the cout, the code looks ok to me.

**Comment by boudoul** [comment]
hi @gouskos , could you please remove the cout ? Thanks

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
+1

**Comment by fioriNTU** [comment]
@deguio @davidlange6 can we have this PR merged, it is very important to monito bad channels in Pixels, in the case the 80X is already closed, can we have #13759 back ported?

**Comment by vanbesien** [comment]
@dmitrijus

**Comment by boudoul** [comment]
hi @davidlange6 , can this PR be merged ? The equivalent in 81X PR #13759  has already been merged some time ago - thanks a lot.

**Comment by davidlange6** [comment]
+1

**Pull request merged**